File: 2614-MaximumCountOfPositiveIntegerAndNegativeInteger/2614-MaximumCountOfPositiveIntegerAndNegativeInteger.js
// Last updated: 10/03/2026, 10:02:48
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumCount = function(nums) {
    let countp =0,countn=0;
    for(let i=0;i<nums.length;i++){
        if(nums[i]==0){
           countp;
            countn;
        }
        else if(nums[i]>0){
            countp++
        }
        else{
            countn++;
        }

    }
    if(countp>countn){
        return countp;
    }
    else{
        return countn;
    }
};
console.log(maximumCount([1,2,3,4,-3,0]))

File: 2614-MaximumCountOfPositiveIntegerAndNegativeInteger/2614-MaximumCountOfPositiveIntegerAndNegativeInteger.cpp
// Last updated: 03/08/2026, 10:07:09
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int pos = 0;
        int neg = 0;

        for (int i = 0; i < nums.size(); i++) {
            
            if (nums[i] > 0) {
                pos++;
            }
            else if (nums[i] < 0) {
                neg++;
            }
        }

        return max(pos, neg);
    }
};